composer.json

```diff
@@ -49,22 +49,22 @@
   },
   "autoload": {
     "psr-4": {
-      "ZfcRbac\\": "src/"
+      "LmcRbac\\": "src/"
     }
   },
   "autoload-dev": {
     "psr-4": {
-      "ZfcRbacTest\\": "test/"
+      "LmcRbacTest\\": "test/"
     }
   },
   "extra": {
     "branch-alias": {
       "dev-master": "2.4-dev",
       "dev-develop": "3.0-dev"
```

Comment:
only "dev-master": "3.0-dev", no?

Reply:
about composer-replace: no, as the namespaces are changed, this is not a drop-in replacement.

Reply:
I think dev-master: 1.1-dev as this package will be released as 1.0.0 soon. but I hardly understand this setting

Reply:
Is it wise to release LmcRbac as v1 instead of v3?

Since this is v3 of ZfcRbac, would it makes more sense for users if it is LmcRbac v3?

I don't use this version of Rbac myself, so it is hard for me to judge.

ZfcRbac v2 was released as LmcRbacMvc v3 because of the breaking change in namespace.